lib/fuse_i.h

```diff
@@ -188,8 +188,11 @@ void cuse_lowlevel_init(fuse_req_t req, fuse_ino_t nodeide, const void *inarg);
 
 int fuse_start_thread(pthread_t *thread_id, void *(*func)(void *), void *arg);
 
-int fuse_session_receive_buf_int(struct fuse_session *se, struct fuse_buf *buf,
-				 struct fuse_chan *ch);
+void fuse_buf_free(struct fuse_buf *buf);
+
+int fuse_session_receive_buf_internal(struct fuse_session *se,
+				      struct fuse_buf *buf,
+				      struct fuse_chan *ch);
 void fuse_session_process_buf_int(struct fuse_session *se,
```

Comment:
Sorry, I missed that process_buf was also `int`. You were being consistent. In that case probably both need to be `int` or `internal`. My preference is the latter though I guess... is there a non-internal variant?

Reply:
@joannekoong had recently changed the internal callers to the non-internal version. Initially looked ok as it simplified the code a bit, but in this PR we see that we need to use the internal functions for libfuse internal calls. The non `_internal` functions are defined the public API (`include/`) and are also exported in `lib/fuse_versionscript`- I'm personally not so happy to have `fuse_session_process_buf`` and `fuse_session_receive_buf` in the public API - no idea what is using that.

I created a new patch for the rename of _int to _internal, instead of having that in the main part. I'm a bit used for fn_int(), my colleagues tend to do the same, but I personally find it also confusing and typically use `_fn` and `__fn`